### PR TITLE
Support horovod above 0.20.x

### LINKED ISSUE #3523: Support horovod 0.21.x

### PR DESCRIPTION
**Pull request by yangw1234**
fix #3523

### CODE REVIEW

File: pyzoo/zoo/orca/learn/horovod/horovod_ray_runner.py [hunk 1]
```diff
@@ -88,9 +92,24 @@ def __init__(self, ray_ctx, worker_cls=None, worker_param=None, workers_per_node
...
         hosts_spec, name_rank_to_id, host_to_size = _hosts_to_hosts_spec(hosts)
-        self.host_alloc_plan = _allocate(",".join(hosts_spec), self.num_nodes)
-        global_rendezv = RendezvousServer(True)
-        global_rendezv_port = global_rendezv.start_server(self.host_alloc_plan)
+
+        major, minor, patch, version_str = get_horovod_version()
+
+        if major == 0 and minor < 19:
+            raise RuntimeError(f"We only support horovod version larger "
+                               f"than 0.19.0, but got {version_str}")
```
Comment (jason-dai) [inline pyzoo/zoo/orca/learn/horovod/horovod_ray_runner.py:100]: newer than

Comment (yangw1234) [reply pyzoo/zoo/orca/learn/horovod/horovod_ray_runner.py:100]: fixed

**Comment by yangw1234** [comment]
http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/5110/ jenkins passed

**Comment by yangw1234** [comment]
also manually tested on horovod 0.21.3

**Pull request merged**